src/_host-overrides.scss

```diff
@@ -24,6 +24,7 @@ a {
   }
 }
 
-a:hover {
+a:hover, a:focus {
```

Comment:
Is this black color an override to stop hosts from doing weird things, or is it actually a design change?